src/properties/propertyowner.cpp

```diff
@@ -324,5 +324,13 @@ const std::string& PropertyOwner::name() const {
     return _name;
 }
 
+const std::vector<std::string> PropertyOwner::tags() const {
+    return _tags;
+}
+
+void PropertyOwner::addTag(std::string tag) {
+    _tags.push_back(tag);
```

Comment:
Better: `_tags.push_back(std::move(tag))` to not rely on the optimizer to remove the copy